src/commcare_cloud/ansible/roles/deploy_hq/tasks/staticfiles_collect.yml

```diff
@@ -1,5 +1,5 @@
 - name: Version static files
-  # Run after any new bower code has been installed.
+  # Run after any new bower (javascript?) code has been installed.
```

Comment:
This get's run on every deploy, not just ones with new JS dependencies, so maybe worth re-wording to remove the furst sentence altogether?